data/mods/ssb/moves.ts

```diff
@@ -3502,69 +3447,49 @@ export const Moves: {[k: string]: ModdedMoveData} = {
 	},
 
 	// Perish Song
-	shiftingrocks: {
-		accuracy: true,
-		basePower: 0,
-		category: "Status",
-		desc: "Entry Hazard. The damage is static for all types, (doesn't factor type effectiveness of rock) and is equal to (7+n)%, n being the number of times a Pokemon takes damage from the hazard. Starts at 0 and caps at 10. After 10 turns, (when the damage reaches 17%) Shifting Rocks explode (disappear) and deal 80-Base Power Rock-type damage to the foe.",
-		shortDesc: "Hazard: ramps up damage then explodes.",
-		name: "Shifting Rocks",
+	trickery: {
+		accuracy: 85,
+		basePower: 100,
+		category: "Physical",
+		desc: "Changes the target's item to something random.",
+		shortDesc: "Changes the target's item to something random.",
+		name: "Trickery",
 		isNonstandard: "Custom",
-		gen: 8,
-		pp: 15,
+		pp: 10,
 		priority: 0,
-		flags: {reflectable: 1},
+		flags: {protect: 1, mirror: 1},
 		onTryMove() {
 			this.attrLastMove('[still]');
 		},
 		onPrepareHit(target, source) {
-			this.add('-anim', source, 'Stealth Rock', target);
-			this.add('-anim', source, 'Stealth Rock', target);
-		},
-		sideCondition: 'shiftingrocks',
-		condition: {
-			// this is a side condition
-			onStart(side) {
-				if (this.field.isTerrain('waveterrain')) {
-					this.add('-message', `Wave Terrain prevented Shifting Rocks from starting!`);
-					return null;
+			this.add('-anim', source, "Amnesia", source);
+			this.add('-anim', source, "Trick", target);
+		},
+		onHit(target, source, effect) {
+			const item = target.takeItem();
+			if (!target.item) {
+				if (item) this.add('-enditem', target, item.name, '[from] move: Trickery', '[of] ' + source);
+				const items: ModdedItemData[] = [];
+				for (const id in Items) {
+					const itemName = Items[id];
+					items.push(itemName);
```

Comment:
Should this check if the item `isNonstandard`? Berserk Gene or Crucibellite might be a bit weird to get.

Reply:
atm I assume we just go full meme with randomizations regardless.

Reply:
I don't think perish song specified any bounds for his custom move, anything from mail to crucibellite is currently fine

Reply:
will talk with perish song, theyre the final verdict on this.